examples/ocean_wind_mixing_and_convection.jl

```diff
@@ -186,7 +186,7 @@ function makeplot!(axs, model)
 end
 
 # Finally, we run the the model in a `while` loop.
-
+old_logger = Logging.global_logger(ModelLogger())
```

Comment:
Do you know about `Logging.with_logger`? You should use that if you save and restore the logger rather than doing it manually like you have here.

Reply:
I saw that function! I would prefer to use it but I didn't see a quick way to plug it into the example. I figured I'd swap the logger at the global level just to see if the macro would work with `ModelLogger`.

Would it be advisable to make a lambda wrapping the model loop and pass it to `Logging.with_logger`?

Reply:
After reading more about `Logging.with_logger` I wrapped the while loop with it.

Reply:
Cool. Admittedly it's annoying to have the extra indentation but other than it looks fine to me.